src/components/pages/navbar/NavbarContainer.jsx

```diff
@@ -44,25 +43,53 @@ const MenuItem = ({ key, to, icon, children, ...props }) => {
 
```

Comment:
Lookin good here! Good idea to change the keys to = path, this makes a lot of sense.